internal/commands/app/new_app_inputs.go

```diff
@@ -94,42 +93,3 @@ func (i *newAppInputs) resolveTemplateID(ui terminal.UI, client realm.Client) er
 
 	return nil
 }
-
-func (i createInputs) resolveInitialTemplateDataSource(ui terminal.UI, dataLakes []dataSourceDatalake, clusters []dataSourceCluster) (interface{}, error) {
```

Comment:
removed this because we only allow 1 cluster to be created with a template app. Since this is the case, we no longer need a `resolve*` function that chooses a template data source because there is only going to be one choice when it gets to this point.